gcs/README.md

```diff
@@ -1,45 +1,16 @@
-# GCS Storage CLI
-A Golang CLI for uploading, fetching and deleting content to/from [Google Cloud Storage](https://cloud.google.com/storage/). 
-This tool exists to work with the [bosh-cli](https://github.com/cloudfoundry/bosh-cli) and [director](https://github.com/cloudfoundry/bosh).
+# GCS Client
 
-This is **not** an official Google Product.
+GCS (Google Cloud Storage) client implementation for the unified storage-cli tool. This module provides Google Cloud Storage operations through the main storage-cli binary.
 
+**Note:** This is not a standalone CLI. Use the main `storage-cli` binary with `-s gcs` flag to access GCS functionality.
 
-## Commands
+For general usage and build instructions, see the [main README](../README.md).
 
-### Usage
-```bash
-gcs-cli --help
-```
-### Upload an object
-```bash
-gcs-cli  -c config.json put <path/to/file> <remote-blob>
-```
-### Fetch an object
-```bash
-gcs-cli  -c config.json get <remote-blob> <path/to/file>
-```
-### Delete an object
-```bash
-gcs-cli  -c config.json delete <remote-blob>
-```
-### Check if an object exists
-```bash
-gcs-cli  -c config.json exists <remote-blob>
-```
+This is **not** an official Google Product.
 
-### Generate a signed url for an object
-If there is an encryption key present in the config, then an additional header is sent
+## GCS-Specific Configuration
 
-```bash
-gcs-cli  -c config.json sign <remote-blob> <http action> <expiry>
-```
-Where:
- - `<http action>` is GET, PUT, or DELETE
- - `<expiry>` is a duration string less than 7 days (e.g. "6h")
-
-## Configuration
-The command line tool expects a JSON configuration file. Run `storage-cli-gcs --help` for details.
+The GCS client requires a JSON configuration file.
```

Comment:
Would be nice to have a configuration example here as well similar to the others. But as this not present in [bosh-gcscli](https://github.com/cloudfoundry/bosh-gcscli) this can be added later.